Sources/MongoSwift/BSON/Document.swift

```diff
@@ -5,6 +5,10 @@ import Foundation
 public class DocumentStorage {
     internal var pointer: UnsafeMutablePointer<bson_t>!
 
+    internal var count: Int {
```

Comment:
nit: let's add a comment so we remember why we put this here.

Reply:
Done.